themes/docs-new/layouts/shortcodes/data_bag_encryption_versions.md

```diff
@@ -1,55 +1,45 @@
--   A recipe can load encrypted data as long as the shared secret is
-    present in a file on the node or is accessible from a URI path
--   Only the values of a data bag item are decrypted; keys are still
-    searchable. The values associated with the `id` key of a data bag
-    item are not encrypted (because they are needed when tracking the
-    data bag item)
-
-For version 1 (default, starting with Chef Client 11.0):
-
--   An encrypted data bag item is written using JSON as the
-    serialization format
--   Base64 encoding is used to preserve special characters in encrypted
-    contents
--   Data is encrypted using AES-256-CBC (as defined by the OpenSSL
-    package in the Ruby Standard Library)
--   A data bag item is encrypted using a random initialization vector
-    each time a value is encrypted, which helps protect against some
-    forms of cryptanalysis
--   Chef Infra Client uses [shared secret
-    encryption](https://en.wikipedia.org/wiki/Symmetric-key_algorithm);
-    an encrypted file can only be decrypted by a node or a user with the
-    same shared secret
--   A recipe can load encrypted data as long as the shared secret is
-    present in a file on the node or is accessible from a URI path
--   Only the values of a data bag item are decrypted; keys are still
-    searchable. The values associated with the `id` key of a data bag
-    item are not encrypted (because they are needed by Chef Infra Client
-    when tracking the data bag item)
-
-For version 2 (available, starting with Chef Client 11.6):
-
--   Same as version 1
--   Can disable version 0 and version 1 data bag item encryption formats
--   Adds Encrypt-then-MAC(EtM) protection
\ No newline at end of file
+|Infra Client version|Encryption v0|Encryption v1|Encryption v2|Encryption v3|
+|:--|:---:|:---:|:---:|:---:|
+|10.x|`R` `W`||||
+|11.0+|`R`|`R` `W`|||
+|11.6+|`R` `D`|`R` `D`|`R` `W`||
+|13.0|`R` `D`|`R` `D`|`R` `D`|`R` `W`|
+
+`R` = read
+`W` = write
+`D` = disable
+
+#### Version 0
+
+Chef Infra Client 10.0+
+
+- Uses YAML serialization format to encrypt data bag items
+- Uses Base64 encoding to preserve special characters
+- Uses AES-256-CBC encryption, as defined by the OpenSSL package in the Ruby Standard Library
+- [Shared secret encryption](https://en.wikipedia.org/wiki/Symmetric-key_algorithm); an encrypted file can only be decrypted by a node or a user with the same shared secret
+- Recipes load encrypted data with access to the shared secret in a file on the node or from a URI path
+- Decrypts only data bag item values. Keys are encrypted but searchable
+- Data bag `id` value is unencrypted for tracking data bag items
+
+#### Version 1
+
+Chef Infra Client 11.0+
+- Version 0
+- Uses JSON serialization format _instead of_ YAML to encrypt data bag items
+- Adds random initialization vector encryption for each value to protect against cryptanalysis
+
+#### Version 2
+
+Chef Infra Client 11.6+
+
+- Version 1
+- Option to disable versions 0 and 1
```

Comment:
I don't think this is actually part of the version 2. It's just a general chef client feature.

Reply:
I think this line exists to let people know they can do it.